codenav/src/utils.c

```diff
@@ -19,7 +19,7 @@
  *  along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "utils.h"
+#include "geanyplugin.h"
```

Comment:
hum, why would we need this if we didn't before?

Reply:
I am not getting that warning if I compile ```codenav/src/utils.c``` from current master.